pkgs/applications/video/openshot-qt/default.nix

```diff
@@ -23,12 +25,9 @@ stdenv.mkDerivation rec {
   installPhase = ''
```

Comment:
If you override `installPhase`, the first line must be `runHook preInstall` and the last line must be `runHook postInstall`.